### DISCUSSION TITLE
curl: (60) SSL certificate problem: certificate is not yet valid (K1C 1.3.3.8)

### DISCUSSION THREAD

**Discussion by alientmpl** [Guilouz/Creality-Helper-Script-Wiki#488] [General]
1) Install Moonraker and Nginx - OK

3) Install Mainsail (port 4409) 

"curl: (60) SSL certificate problem: certificate is not yet valid
More details here: https://curl.haxx.se/docs/sslcerts.html

curl failed to verify the legitimacy of the server and therefore could not
establish a secure connection to it. To learn more about this situation and
how to fix it, please visit the web page mentioned above."

any help ?

**Comment by TheWebMachine** [comment]
Did you confirm your timezone and such is correct as per the wiki?

**Reply by alientmpl** [reply]
thank you, the wrong date was probably the problem
